Software/scan_key/AuthenticationSQL.py

```diff
@@ -62,8 +61,34 @@ def twoFactorAuth(id, pin, cursor):
         return(False) #if pin is invalid, then not allowed
 
 def getID(idString, cursor):
-    cursor.execute("SELECT CUID, pin FROM USER WHERE t1String = " + idString) #getting user w/ the id
+    cursor.execute("SELECT CUID FROM USER WHERE t1String = " + idString) #getting user w/ the id
     data = cursor.fetchone() #fetching data into array
-
     cuid = data[0] #and this is the cuid of the person with the id string
     return cuid
+
+def getMachineID():
+    currMachineID = subprocess.check_output(['hostname'])
+    currMachineID = currMachineID.strip()
+    currMachineID = (currMachineID).decode('UTF-8')
+    return(currMachineID)
```

Comment:
Do our style conventions call for return() or just return? All the functions below this use return.